src/ParcelRegistry.Projections.BackOffice/BackOfficeProjections.cs

```diff
@@ -0,0 +1,60 @@
+    using Parcel;
+    using Parcel.Events;
+
+    public class BackOfficeProjections : ConnectedProjection<BackOfficeProjectionsContext>
+    {
+        public BackOfficeProjections(IDbContextFactory<BackOfficeContext> backOfficeContextFactory)
+        {
+            When<Envelope<ParcelAddressWasAttachedV2>>(async (_, message, cancellationToken) =>
+            {
+                await using var backOfficeContext = await backOfficeContextFactory.CreateDbContextAsync(cancellationToken);
+                await backOfficeContext.AddIdempotentParcelAddressRelation(
+                    new ParcelId(message.Message.ParcelId),
+                    new AddressPersistentLocalId(message.Message.AddressPersistentLocalId),
+                    cancellationToken);
+            });
+
+            When<Envelope<ParcelAddressWasDetachedV2>>(async (_, message, cancellationToken) =>
+            {
+                await using var backOfficeContext = await backOfficeContextFactory.CreateDbContextAsync(cancellationToken);
+                await backOfficeContext.RemoveIdempotentParcelAddressRelation(
+                    new ParcelId(message.Message.ParcelId),
+                    new AddressPersistentLocalId(message.Message.AddressPersistentLocalId),
+                    cancellationToken);
+            });
+
+            When<Envelope<ParcelAddressWasDetachedBecauseAddressWasRejected>>(async (_, message, cancellationToken) =>
+            {
+                await using var backOfficeContext = await backOfficeContextFactory.CreateDbContextAsync(cancellationToken);
+                await backOfficeContext.RemoveIdempotentParcelAddressRelation(
+                    new ParcelId(message.Message.ParcelId),
+                    new AddressPersistentLocalId(message.Message.AddressPersistentLocalId),
+                    cancellationToken);
+            });
+
+            When<Envelope<ParcelAddressWasDetachedBecauseAddressWasRetired>>(async (_, message, cancellationToken) =>
+            {
+                await using var backOfficeContext = await backOfficeContextFactory.CreateDbContextAsync(cancellationToken);
+                await backOfficeContext.RemoveIdempotentParcelAddressRelation(
+                    new ParcelId(message.Message.ParcelId),
+                    new AddressPersistentLocalId(message.Message.AddressPersistentLocalId),
+                    cancellationToken);
+            });
+
+            When<Envelope<ParcelAddressWasDetachedBecauseAddressWasRemoved>>(async (_, message, cancellationToken) =>
+            {
+                await using var backOfficeContext = await backOfficeContextFactory.CreateDbContextAsync(cancellationToken);
+                await backOfficeContext.RemoveIdempotentParcelAddressRelation(
+                    new ParcelId(message.Message.ParcelId),
+                    new AddressPersistentLocalId(message.Message.AddressPersistentLocalId),
+                    cancellationToken);
+            });
```

Comment:
These shouldn't be done, cause these come via the consumer. Which won't have the "transaction" problem as in the api. But it's also not forbidden to put it here, an extra check :)